src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/EnableAzureRmAlias.cs

```diff
@@ -89,6 +98,10 @@ public void EnableLocalAliases(string[] modulesToEnable, Dictionary<string, obje
                     }
                 }
             }
+            if (Scope != null && Scope.Equals("Process"))
+            {
+                powershell.Invoke();
```

Comment:
I think we can do this a little more cleanly using PSCmdlet.InvokeCommand.InvokeScript or by building a script block using PSCmdlet.InvokeCommand and doign the same.  Would have to make sure this is available in 5.1 S.M.A as well, though.